### PR TITLE
add xmlutils, containing `squashed_element_text` & some whitespace-normalizing functions

### PR DESCRIPTION
**Pull request by risicle**
These are extremely useful in tests and it would be nice to have them available project-wide.

### CODE REVIEW

File: dmutils/xmlutils.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import re
+
+
+def squashed_element_text(element):
```
Comment (allait) [inline dmutils/xmlutils.py]: [`.text_content()`](http://lxml.de/lxmlhtml.html) seems to do exactly this

Comment (risicle) [inline dmutils/xmlutils.py]: God bleeding damn it.

**Comment by risicle** [comment]
At least this PR taught me that lxml on my local machine and on travis do _not_ behave _quite_ alike.

**Comment by allait** [comment]
@risicle whitespace normalisation seems quite useful, I think we can merge that without the text function

**Comment by risicle** [comment]
I'm not sure about that - it's so trivial it doesn't seem worth it, especially seeing as I'm not even sure it is standards-compliant. Also think `lxml.html` might have some utility functions for this

**Pull request closed without merging**